docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,17 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+[IMPORTANT]
+====
+Using a database to store transaction logs has several benefits, but in comparison with using the file system to store the logs, you might notice a reduction in performance.
+Storing transaction logs into a database is slower, creates bigger overhead, and requires higher maintenance.
+Use this approach when prioritizing consistency over throughput.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transacion-manager.object-store.<property>` properties, where <property> can be:
+In addition, this methodology is only relevant for eXtended Architecture (XA) scenarios.
+However, use XA in cloud-native applications only if explicitly required by your use case.
```

Comment:
Two things here:
* 'XA' alone doesn't sound clear to me...'XA transactions' is better
* 'only if explicitly required by your use case' could be replaced with 'only after careful evaluations'

Probably, we would like to add a sentence here to be more specific why XA transactions need 'careful evaluations' in cloud-native applications. In particular, Narayana (i.e. the framework that handles XA transactions) needs a stable storage, a unique (and reusable) node identifier and a stable IP to run XA transactions. When the JDBC object store is used, one of these requirements is fulfilled but users still need to consider strategies to satisfy the other two requirements. More details [here](https://jbossts.blogspot.com/2022/04/narayana-on-cloud-part-1.html)